src/actions/surveys.js

```diff
@@ -20,15 +20,16 @@ export function fetchSurveysMetadata() {
   return (dispatch) => {
     // TODO(ayushbhagat): Make the get request to get the metadata for the surveys, instead of
     // getting it from the mock data.
-    dispatch(changeSurveysMetadata(Immutable.fromJS(mockSurveysMetadata)));
+    dispatch(changeSurveysMetadata(mockSurveysMetadata));
```

Comment:
Can you add a comment in the commit message as to why its not longer Immutable when merging the PR ?